[jnyz-app/server]2024/05/04 - 13:10:20.471	[31merror[0m	/home/amos/repo/jnyz-app/server/api/v1/data/data_deposit_sub.go:46	æŸ¥è¯¢å¤±è´¥!	{"error": "record not found"}
[jnyz-app/server]2024/05/04 - 16:35:25.268	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 16:36:34.932	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 16:54:50.882	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 16:55:33.237	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 17:09:38.584	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 17:09:42.423	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 17:10:29.931	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 17:13:58.042	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 17:14:28.529	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 18:17:11.939	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 18:21:52.466	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 18:22:37.958	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 18:23:15.412	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 18:33:34.636	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 18:34:19.157	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 19:57:12.045	[31merror[0m	/home/amos/repo/jnyz-app/server/api/v1/data/data_deposit_sub.go:46	æŸ¥è¯¢å¤±è´¥!	{"error": "record not found"}
[jnyz-app/server]2024/05/04 - 19:59:32.597	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 20:08:51.938	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 20:26:49.694	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 20:26:56.884	[31merror[0m	/home/amos/repo/jnyz-app/server/api/v1/data/data_deposit_sub.go:45	æŸ¥è¯¢å¤±è´¥!	{"error": "record not found"}
[jnyz-app/server]2024/05/04 - 20:27:07.984	[31merror[0m	/home/amos/repo/jnyz-app/server/api/v1/data/data_deposit_sub.go:45	æŸ¥è¯¢å¤±è´¥!	{"error": "record not found"}
[jnyz-app/server]2024/05/04 - 20:30:58.211	[31merror[0m	/home/amos/repo/jnyz-app/server/api/v1/data/data_deposit_sub.go:45	æŸ¥è¯¢å¤±è´¥!	{"error": "record not found"}
[jnyz-app/server]2024/05/04 - 20:31:15.731	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 20:33:36.337	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 20:39:00.677	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 20:59:10.996	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 21:00:01.807	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 21:15:44.314	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 21:16:00.740	[31merror[0m	/home/amos/repo/jnyz-app/server/api/v1/data/data_deposit_sub.go:45	æŸ¥è¯¢å¤±è´¥!	{"error": "record not found"}
[jnyz-app/server]2024/05/04 - 21:17:16.874	[31merror[0m	/home/amos/repo/jnyz-app/server/api/v1/data/data_deposit_sub.go:45	æŸ¥è¯¢å¤±è´¥!	{"error": "record not found"}
[jnyz-app/server]2024/05/04 - 21:19:45.611	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 21:19:55.936	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:55	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/05/04 - 22:04:47.429	[31merror[0m	/home/amos/repo/jnyz-app/server/api/v1/data/data_loan_sub.go:46	æŸ¥è¯¢å¤±è´¥!	{"error": "record not found"}
